USF-2278: initial draft of changes for removing references to gdrive and sharepoint (#277)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/all/linking.mdx

description: Learn how to programmatically manage localized links.
---

> The best practice for localizing the text in a URL is to maintain this text within the content. However, you can use functions defined in the `scripts/scripts.js` file to localize links programmatically.

Reviewer: ```suggestion
The best practice for localizing URL text is to store it within the content. Alternatively, you can use functions from `scripts/scripts.js` to localize links programmatically."
```

## rootLink

> The `rootLink` function prepends each language's root path to the link it receives. Use it within a block to localize links coming from a drop-in for loading scripts, styles, or links to other pages within a drop-in. This approach is necessary for maintaining consistency across different languages and store views.

Reviewer: ```suggestion
The rootLink function prepends the appropriate language root path to a given link. Use it within a block to localize links from a drop-in for loading scripts, styles or links to other pages within a drop-in. This approach ensures consistency across languages and store views.
```
